src/styles/blocks/calendar.scss

```diff
@@ -1,45 +1,43 @@
 .calendar {
   display: flex;
-  flex-flow: row wrap;
-  gap: $gap;
-  width: $cell-side * 7 + $gap * 6 + $calendar-padding * 2;
-  padding: $calendar-padding;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 70px 10px;
 
   &__day {
     display: flex;
     justify-content: center;
     align-items: center;
-    @include squareCell($cell-side);
-    background: #eee;
+    width: #{$container-day-width};
+    height: #{$container-day-height};
```

Comment:
You can write your variables without the #{variable-name}, you can use $variable-name instead.